packages/ember-simple-auth/lib/simple-auth/utils/base64.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Please don't add decode as we don't need it and it just adds unnecessary weight to the library.